Source/WebKitLegacy/morphos/WebProcess.cpp

```diff
@@ -744,7 +758,11 @@ void WebProcess::dumpWebCoreStatistics()
     // Get WebCore memory cache statistics
     getWebCoreMemoryCacheStatistics(ss);
 	
+	#if OS(AMIGAOS)
+	dprintf(0, ss.release().utf8().data());
```

Comment:
I am not sure about this line. If anyone have a better way to do it, please let me know.

Reply:
It might be an idea to switch the FD from 0 to 1, since 0 represents STDIN. If this is used to report errors, then switch it to 2 - or you can probably just use "stdout" / "stderr" directly.